packages/components/src/border-control/types.ts

```diff
@@ -119,12 +119,11 @@ export type BorderControlProps = ColorProps &
 		 */
 		withSlider?: boolean;
 		/**
-		 * Start opting into the larger default height that will become the
-		 * default size in a future version.
+		 * Size of the control.
 		 *
-		 * @default false
+		 * @default default
```

Comment:
```suggestion
		 * @default 'default'
```
I believe the README also needs to be updated